pkg/features/kube_features.go

```diff
@@ -743,9 +743,9 @@ const (
 	// owner: @derekwaynecarr
 	// alpha: v1.20
 	// beta: v1.22
-	//
+	// GA: v1.32
 	// Enables kubelet support to size memory backed volumes
-	SizeMemoryBackedVolumes featuregate.Feature = "SizeMemoryBackedVolumes"
+	SizeMemoryBackedVolumes featuregate.Feature = "SizeMemoryBackedVolumes" // remove in 1.34
```

Comment:
35 - the new version-compat requires 3-deep memory, so 34 has to be able to mimic 33, 32, 31.  In 31 this was beta.

Reply:
where can I read how this version compat works? We are locking the feature gate to default so it is not possible to disable the fearure gate. What would version compat do?

Reply:
Also, we will need to preserve all the tests too? Like ones that disable the feature gate?

I thought compat mode was only for api server

Reply:
I updated this to say remove for 1.35.

Reply:
@jpbetz and @Jefftree were discussion how best to update docs.  The compat is IIUC, fotr the whole system - APIs and controllers.

Reply:
https://github.com/kubernetes/kubernetes/pull/126981#discussion_r1799432320